addons/binding/org.openhab.binding.netatmo/.classpath

```diff
@@ -12,5 +11,6 @@
 	<classpathentry kind="lib" path="lib/json-20131018.jar"/>
 	<classpathentry kind="lib" path="lib/commons-codec-1.8.jar"/>
 	<classpathentry kind="lib" path="lib/gson-2.3.1.jar"/>
```

Comment:
I have some issues with this include. Isn't the gson library part of the ESH framework since a while.